src/services/DfusionService.ts

```diff
@@ -67,6 +67,7 @@ export interface TokenDto {
   decimals: number
   address: string
   known: boolean
+  forceAddressDisplay?: boolean
```

Comment:
If you end up calling this `deprecated`, I would make the field mandatory. Just false by default

Reply:
The logic for the `if` that defines `tokenParam` and `tokenLabel ` can be the same

```js
if (token.deprecated) {
    tokenParam = token.address
    tokenLabel = token.address
  }  else ...
```

Reply:
> If you end up calling this deprecated, I would make the field mandatory. Just false by default
Makes sense.

Regarding the field name, replied in the other thread.